pixijs/extension-scripts
# feat: Improve test convention, update to v8

### Changes

* Convert `./test` folder to Jest-convention `src/__tests__`, encourage the co-location of tests and source
* Adds `tsconfigTypes` to optionally define a configuration for generating types
* No longer adds `test` folder to the files to lint
* Adds support for importing `*.wgsl` files as string in Jest
* Fix: ensure tsconfig is set (now required for `eslint@9`)
* Update test code to `pixi.js@8` and example to use importmap
* Add additional tsconfig options for more strictness
* Removes triple-slash type hack, you can now use `declare global` in src and that will be outputted instead
